meta-mender-core/classes/mender-setup.bbclass

```diff
@@ -230,13 +230,7 @@ MENDER_DATA_PART_LABEL ??= "data"
 # --------------------------- END OF CONFIGURATION -----------------------------
 
 IMAGE_INSTALL:append = " mender-client"
-IMAGE_CLASSES += "mender-artifactimg \
-                  mender-artifact-uefi-capsule \
-                  mender-bootimg \
-                  mender-dataimg \
-                  mender-datatar
-                  mender-part-images \
-                  mender-ubimg"
+IMAGE_CLASSES += "mender-artifactimg mender-artifact-uefi-capsule mender-bootimg mender-dataimg mender-datatar mender-part-images mender-ubimg"
```

Comment:
The line breaks were wrong, `mender-datatar` was missing a backslash. Can we still use the line-broken one?